### ISSUE TITLE
Homework Week 2

### ISSUE THREAD

**Issue by ptums** [aishadurrani/Homework-Week-2.2#1]
Hey Aisha,

Awesome on the job on the homework you hit all the objectives in the assignment and your portfolio looks great! I do have one concern and it has to do with the weekonehw file. You have a composer.json and an index.php file in there. Those two files are used to connect to heroku. Just for the sake of saftey I'd remove index.php and composer.json from weekonehw file. In this case you are good, but in the future it may mess something up. Other than that you did really well on the homework :)

**Comment by aishadurrani** [comment]
Thank you so much Peter 😊 

Your talking about the files that copied into that folder right ? 

And another thing I wanted to ask is will it make a change in the original week one homework ? 

Aisha Durrani

**Comment by ptums** [comment]
Yep. At this point I wouldn't worry about week one homework. You can go as far as to take out the weekone folder out of your working folder and put it in a safe plac.e
